File: backend/API/tg_bots/Model/NotificationGetter.py
from Model.TgAccount import TgAccount


class NotificationGetter(TgAccount):
    restaurant_id: int
    is_blocked: bool

    def __init__(self,
                 id: int,
                 username: str,
                 restaurant_id: int,
                 is_blocked: bool = False,
                 **kwargs):
        super().__init__(id=id,
                         username=username,
                         restaurant_id=restaurant_id,
                         is_blocked=is_blocked,
                         **kwargs)

File: backend/Domain/Models/NotificationGetter.cs
using System.Text.Json.Serialization;
using Domain.DataBaseAccess;

namespace Domain.Models;

public class NotificationGetter(string username, long? id, long restaurantId, bool isBlocked)
    : TgAccount(id, username), ITableDataBase
{
    [JsonPropertyName("restaurant_id")] public long RestaurantId { get; init; } = restaurantId;

    [JsonPropertyName("is_blocked")] public bool IsBlocked { get; init; } = isBlocked;
}